src/message.ts

```diff
@@ -613,9 +721,44 @@ export class Message implements Sayable {
   // }
 
   /**
-   * @todo document me
+   * Reply a text message to the sender.
+   *
+   * @param {string} text
+   * @param {(Contact | Contact[])} [replyTo]
+   * @returns {Promise<any>}
+   * @example
+   * ```ts
+   * const bot = Wechaty.instance()
+   * bot
+   * .on('message', async m => {
+   *   if (/^(ding|ping|bing|code)$/i.test(m.content())) {
+   *     await m.say('dong')
+   *     console.log('Bot REPLY: dong')
+   *   }
+   * })
+   * ```
+   * @see https://github.com/Chatie/wechaty/blob/master/example/ding-dong-bot.ts
    */
   public say(text: string, replyTo?: Contact | Contact[]): Promise<any>
+
+  /**
+   * Reply a media message to the sender.
+   *
+   * @param {MediaMessage} mediaMessage
+   * @param {(Contact | Contact[])} [replyTo]
+   * @returns {Promise<any>}
+   * @example
+   * ```ts
+   * const bot = Wechaty.instance()
+   * bot
+   * .on('message', async m => {
+   *   if (/^(ding|ping|bing|code)$/i.test(m.content())) {
```

Comment:
The same as above: `if` is no need to be so complicated at here. If want to keep it, just a `ding` will be enough.